### ISSUE TITLE
Cannot compile project

### ISSUE THREAD

**Issue by hubca** [GIVESocialMovement/sbt-vuefy#36]
Looks like an interesting project but I cannot compile the test project. The error reads -  

> error: not found: value SbtVuefy

This has brought up other errors on the build as doesn't recognise `Assets` or `VueKeys`. So it seems that the SBT files aren't available for some reason. The build/spec is the same of course, i.e. `Scala 2.12.8 / SBT 1.2.8 / Play 2.7.3`. I've read [this similar issue](https://github.com/GIVESocialMovement/sbt-vuefy/issues/7) - but it didn't help. To note: I used `npm install` instead of `yarn install` as I wasn't able to install `yarn` with the error - 

> 'yarn' is not recognized as an internal or external command, operable program or batch file.

Hopefully that's not an issue. Any help/guidance is greatly appreciated!
_

**Comment by tanin47** [comment]
Using yarn or npm shouldn't matter here.

Is it possible to share your project somewhere? Is it the exact test project here: https://github.com/GIVESocialMovement/sbt-vuefy/tree/master/test-play-project

**Comment by hubca** [comment]
Actually I had to grab a new Play seed and then copy across files into the project - as when setting-up the test project; the SBT functionality were all showing errors and I could not get past them. I will create a repo for the new Play seed with sbt-vuefy and invite you so that you can see. Thanks_

**Comment by hubca** [comment]
@tanin47 I sent you an invite to the repo - any thoughts?

**Comment by tanin47** [comment]
Yes, I've never used `projects/Common.scala` before. I'm not sure how it works exactly.

But, if you put `Assets / VueKeys.vuefy / VueKeys.prodCommands := Set("stage"),` in `build.sbt` directly, does it work?

**Comment by hubca** [comment]
Initially I did try to build the `test-project` exactly as it is but didn't have any success. But strangely I have just put the settings into the `build` and it is thankfully working. The `projects/Common.scala` file was only in order to reuse the settings as I have a sub-project in the application (https://www.playframework.com/documentation/2.5.x/SBTSubProjects) - so it is only creating a variable for the `settings` and therefore I'm not sure why this was an issue. Anyway the next challenge is to try get something pre-rendered on the server-side - like [this react project](https://github.com/ssorallen/react-play). Do you have any ideas on this? Thanks again - it looks tight!
